java/client/src/org/openqa/selenium/devtools/network/model/ResourceTiming.java

```diff
@@ -219,4 +249,101 @@ public Double getReceiveHeadersEnd() {
   public void setReceiveHeadersEnd(Double receiveHeadersEnd) {
     this.receiveHeadersEnd = receiveHeadersEnd;
   }
+
+  private static ResourceTiming fromJson(JsonInput input) {
+
+    input.beginObject();
+
+    Double requestTime = null;
+
+    Double proxyStart = null;
+
+    Double proxyEnd = null;
+
+    Double dnsStart = null;
+
+    Double dnsEnd = null;
+
+    Double connectStart = null;
+
+    Double connectEnd = null;
+
+    Double sslStart = null;
+
+    Double sslEnd = null;
+
+    Double workerStart = null;
+
+    Double workerReady = null;
+
+    Double sendStart = null;
+
+    Double sendEnd = null;
+
+    Double pushStart = null;
+
+    Double pushEnd = null;
+
+    Double receiveHeadersEnd = null;
+
+    while (input.hasNext()) {
+      switch (input.nextName()) {
+        case "requestTime":
```

Comment:
Nit: can we please have these in alphabetical order?

Reply:
done